lib/flat_search_web/live/page_live.ex

```diff
@@ -18,6 +18,73 @@ defmodule FlatSearchWeb.PageLive do
       |> Filters.change_filter(params)
       |> Map.put(:action, :insert)
 
-    {:noreply, assign(socket, changeset: changeset)}
+    params = Enum.map(params, fn {key, value} -> {key, insensitive_string(value)} end)
+
+    {:noreply,
+     assign(socket, changeset: changeset, flats: Flats.get_flats_by(params), filters: params)}
+  end
+
+  @impl true
+  def handle_info({:flat_created, flat}, socket) do
+    case flat_fulfills_filters?(socket.assigns.filters, flat) do
+      true ->
+        {:noreply, assign(socket, flats: [flat] ++ socket.assigns.flats)}
+
+      false ->
+        {:noreply, assign(socket, flats: socket.assigns.flats)}
```

Comment:
just {:noreply, socket} would do the job, you don't need to assign anything since nothing changes in this case, right?

Reply:
Thats great, this one seemed so inefficient, I forgot to go back to it